fastlane/lanes/release.rb

```diff
@@ -379,53 +390,45 @@
   # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
   #
   desc 'Trigger the final release build on CI'
-  lane :finalize_release do |options|
+  lane :finalize_release do |skip_confirm: false|
     UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
 
-    ensure_git_branch_is_release_branch
-
-    # Verify that there's nothing in progress in the working copy
+    ensure_git_branch_is_release_branch!
     ensure_git_status_clean
 
-    skip_user_confirmation = options[:skip_confirm]
-
     UI.important("Finalizing release: #{release_version_current}")
-    UI.user_error!('Aborted by user request') unless skip_user_confirmation || UI.confirm('Do you want to continue?')
-
-    git_pull
+    UI.user_error!('Aborted by user request') unless skip_confirm || UI.confirm('Do you want to continue?')
 
-    check_all_translations(interactive: skip_user_confirmation == false)
+    check_all_translations(interactive: skip_confirm == false)
 
     download_localized_strings_and_metadata(options)
-    lint_localizations(allow_retry: skip_user_confirmation == false)
+    lint_localizations(allow_retry: skip_confirm == false)
 
     bump_build_codes
 
-    unless skip_user_confirmation || UI.confirm('Ready to push changes to remote and trigger the release build?')
-      UI.message("Terminating as requested. Don't forget to run the remainder of this automation manually.")
-      next
-    end
+    UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.") unless skip_confirm || UI.confirm('Ready to push changes to remote and trigger the release build?')
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/wordpress-mobile/WordPress-iOS/blob/3ea5ce7fb46853fc78e2b0ca2196de39e75cd487/fastlane/lanes/release.rb#L409"></span>Style/IfUnlessModifier: Modifier form of <code>unless</code> makes the line too long.</td>
    </tr>
  </tbody>
</table>